neurom/fst/_neuritefunc.py

```diff
@@ -307,17 +307,40 @@ def partition_asymmetries(neurites, neurite_type=NeuriteType.all):
                              neurite_filter=is_type(neurite_type)))
 
 
-def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
-    '''Remote bifurcation angles in a collection of neurites'''
+def partition_pairs(neurites, neurite_type=NeuriteType.all):
+    '''Partition pairs at bifurcation points of a collection of neurites'''
```

Comment:
Can you had the definition of a partition pair in the docstring :

The partition pair is the number of bifurcations in the two daughter subtrees at each branch point.

Reply:
True! Will correct this documentation.